### DISCUSSION TITLE
Checkboxes in tableview?

### DISCUSSION THREAD

**Discussion by NomadicDeveloper22** [palexdev/MaterialFX#150] [Archived]
Does materialfx allow/support a way to combine MFXPaginatedTableView with MFXCheckListView? I want the pagination and ability to easily filter data with the tableview, but also the ability to have each row check-able like with checklistview

**Comment by palexdev** [comment]
Yes it is possible, you have to create custom table rows and cells.
The simplest way I think is to have an empty column which creates row cells with just a combo box.
Then the issue is telling the row to use the combo box for the selection instead of the mouse

At the moment though the process is quite cumbersome as I overlooked the implementation
I'll make it better in a future version
